lib/metadata_placement.js

```diff
@@ -126,12 +127,55 @@ function MetadataPlacement(opts) {
 
             // Establish a buckets-mdapi connection to each shard
             self.log.info('creating buckets-mdapi clients');
+
+            // Create a filtered logger for buckets-mdapi to
+            // reduce user error noise
```

Comment:
I asked a similar question above:  Why reducing the error noise? There's better text here (I hope I didn't miss that better text above?), that suggests we don't want S3 clients denying-service on the server side with excessive messages. Is that the reason we suppress?